crates/rpc/rpc-engine-api/src/capabilities.rs

```diff
@@ -19,6 +19,21 @@ pub const CAPABILITIES: &[&str] = &[
     "engine_getPayloadBodiesByRangeV1",
     "engine_getBlobsV1",
 ];
+/// The list of all supported Engine capabilities available over the engine endpoint.
```

Comment:
```suggestion
/// The list of all supported Engine capabilities available over the engine endpoint.
///
/// Spec: <https://specs.optimism.io/protocol/exec-engine.html>
```
nitpick, is nice to have link to specs where possible